src/main/java/it/pagopa/ecommerce/payment/instruments/domain/valueobjects/Psp.java

```diff
@@ -13,13 +13,13 @@
 @Getter
 public class Psp implements Serializable {
```

Comment:
I was thinking about the naming of this class: what do you think about `PspSummary`? The thing that doesn't sit well with me it's the fact that we have both an aggregate and a value object with the same name, but maybe I'm just worrying too much over nothing

Reply:
That's a good point, maybe we can remove the value object, it was defined before the domain model refactoring. Now we have separate collection for PaymentInstruments and PSPs. Should we do it in this PR or another one?

Reply:
If it's not too messy I think it's worth removing the value object in this PR. Although the threshold of "messy" should be pretty tight: there's already a lot going on in this PR and I think it's better to keep this clean that doing too much at once